apps/core-angular-universal/package.json

```diff
@@ -11,48 +11,47 @@
     "dev:ssr": "ng run core-angular-universal:serve-ssr",
     "serve:ssr": "node dist/core-angular-universal/server/main.js",
     "prerender": "ng run core-angular-universal:prerender",
-    "postinstall": "ngcc --create-ivy-entry-points && node ../../scripts/ngcc-clean"
+    "postinstall": "ngcc && node ../../scripts/ngcc-clean"
```

Comment:
The updating `ngcc --create-ivy-entry-points` to `ngcc` fixes the lock timeout issues. This may also fix the timeouts we have seen in the CI. The flag is no longer needed as of ng 11 ngcc can run in parallel without blocking on the creation of the entrypoints.